### PR TITLE
144 basic preferences

### PR DESCRIPTION
**Pull request by timbl**
A branch to re-introduce basic preferences pane.

### CODE REVIEW

File: dashboard/basicPreferences.source.ts [hunk 1]
```diff
@@ -21,50 +26,89 @@ export const basicPreferencesPane: PaneDefinition = {
...
-  ui:label "Expand image URLs inline".
-:newestFirst a ui:BooleanField; ui:property solid:newestFirst;
-  ui:label "Newest messages at the top".
+# Since these options are opt-in, it is a bit strange to have new users opt in
+# That they are new users - also we do not use this class for anything specific
+# yet
+# solid:NewUser a rdfs:Class;
+#  rdfs:label "new user"@en;
+#  rdfs:comment """A person who might use a Solid-based system who has low
+#  level of familarity with technical details.""";
```
Comment (TallTed) [inline dashboard/basicPreferences.source.ts]:
```suggestion
#  level of familiarity with technical details.""";
```

File: dashboard/basicPreferences.source.ts [hunk 2]
```diff
@@ -21,50 +26,89 @@ export const basicPreferencesPane: PaneDefinition = {
...
-:developerUser a ui:BooleanField; ui:property solid:developerUser;
-  ui:label "Expand image URLs inline".
-:newestFirst a ui:BooleanField; ui:property solid:newestFirst;
-  ui:label "Newest messages at the top".
+# Since these options are opt-in, it is a bit strange to have new users opt in
+# That they are new users - also we do not use this class for anything specific
+# yet
+# solid:NewUser a rdfs:Class;
+#  rdfs:label "new user"@en;
+#  rdfs:comment """A person who might use a Solid-based system who has low
```
Comment (TallTed) [inline dashboard/basicPreferences.source.ts]:
```suggestion
#  rdfs:comment """A person who might use a Solid-based system who has a low
```

File: dashboard/ontology-extra.ttl [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+@prefix : <#>.
+
+solid:User a rdfs:Class;
+  rdfs:label "user"@en, "utilisateur"@fr;
+  rdfs:comment """Any person who might use a Solid-based system""";
+  rdfs:subClassOf foaf:Person, schema:Person, vcard:Indivudual.
+
+solid:NewUser a rdfs:Class;
+  rdfs:label "new user"@en;
+  rdfs:comment """A person who might use a Solid-based system who has low
```
Comment (TallTed) [inline dashboard/ontology-extra.ttl]:
```suggestion
  rdfs:comment """A person who might use a Solid-based system who has a low
```

File: dashboard/ontology-extra.ttl [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  level of familarity with technical details.""";
```
Comment (TallTed) [inline dashboard/ontology-extra.ttl]:
```suggestion
  level of familiarity with technical details.""";
```

**Comment by megoth** [comment]
I realized that this PR is probably outdated because of https://github.com/solid/solid-panes/pull/176.

@timbl are you ok with closing this PR in favor of https://github.com/solid/solid-panes/pull/176 ? (I'll close it for now, and reopen it if you disagree.)

@TallTed I'll make sure that your typo-fixes are transferred to that PR.

**Pull request closed without merging**